src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -21,8 +21,8 @@
 
 {% block money_widget -%}
     <div class="input-group">
-        {% set prepend = '{{' == money_pattern[0:2] %}
-        {% if not prepend %}
+        {% set prepend = '{{' != money_pattern[0:2] %}
```

Comment:
I love the readability of the `starts with` Twig operator. If you agree, you could change this code as follows:

```twig
{% block money_widget -%}
    <div class="input-group">
        {% set append = money_pattern starts with '{{' %}
        {% if not append %}
            <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
        {% endif %}
        {{- block('form_widget_simple') -}}
        {% if append %}
            <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
        {% endif %}
    </div>
{%- endblock money_widget %}
```